### DISCUSSION TITLE
トラックボールが突然効かなくなった

### DISCUSSION THREAD

**Discussion by Thai7Ken** [Yowkees/keyball#699] [Q&A]
Keyball 39を最近し始めた初心者です。
ある時、突然トラックボールが動かなくなりました。
他にキーは正常でしたが、接続していたCケーブルを差し直すと、復活しました。
その後は使用できましたが、はんだ付け等で見直すところがあれば教えていただきたいです。
どうぞよろしくお願いいたします。

**Comment by Yowkees** [comment]
Keyball39をご愛用いただき、誠にありがとうございます。

再起動で復活できたということで、トラックボール基板とコンスルーの接続が不安定なのかも知れません。
トラックボール基板がL字コンスルーの奥まで挿さっているかご確認してみてください。

以上、よろしくお願いいたします。

**Reply by Thai7Ken** [reply]
Yowkees様
返信いただきありがとうございました。
確実に差し込むと、その後不良は発生していませんので、解決したかと考えます。
